containerfixture/containerfixture.go

```diff
@@ -130,7 +144,8 @@ func cleanup(t *testing.T, podName string) {
 	}
 }
 
-func WithServiceContainer(t *testing.T, ServiceManifest string, testFunc func(FixtureInfo)) {
+func WithServiceContainer(t *testing.T, serviceName, serviceDirPath, ServiceManifest string, testFunc func(FixtureInfo)) {
+	buildContainer(t, serviceName, serviceDirPath)
```

Comment:
The one benefit of `kube play` is that it can skip building if its not necessary, can this do that too?

Reply:
did you check wither the fact that we have already built all the image layers means that the build is by-and-large skipped automatically?

Reply:
the build process will be mostly reusing the existing layers,  and it won't rebuild them. so the whole build itself won't be skipped, but it'll be very fast.

Reply:
did you see that in action or are you just assuming it would happen?

Reply:
I'm building the images and running the containers over and over again; Yes, I saw that in action.